lib/active_stash/collection_proxy.rb

```diff
@@ -67,6 +65,19 @@ def client
         )
       end
 
+      def indexes(collection)
+        collection.instance_variable_get("@indexes")
+      end
```

Comment:
If we are going to do this properly we should probably expose `indexes` in the lower level client.

Reply:
I agree that should be exposed via an official interface in `cipherstash-client`